server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/routes/ErrorRoutesTest.java

```diff
@@ -95,6 +95,6 @@ public void defineJsonExtractExceptionShouldReturnBadRequestJsonFormat() throws
             .body("statusCode", equalTo(BAD_REQUEST_400))
             .body("type", equalTo(INVALID_ARGUMENT.getType()))
             .body("message", equalTo("JSON payload of the request is not valid"))
-            .body("cause", equalTo("java.lang.Exception: error while parsing object"));
+            .body("cause", equalTo("com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'a': was expecting ('true', 'false' or 'null')\n at [Source: a non valid JSON; line: 1, column: 2]"));
```

Comment:
That's way better but why it's not the "message" field ? and why are we leaking the fact it throws a Java exception ?

Reply:
I see `cause` field from `ErrorDetail` is used in many places to generate JSON error messages, seem like we did accept to reveal `cause` (`Throwable.getMessage()`). 
And `message` I think is designed to contain custom friendly reading message, not `Throwable.getMessage()` - suppose to be `cause` field

Reply:
To me, `Unrecognized token 'a': was expecting ('true', 'false' or 'null')\n at [Source: a non valid JSON; line: 1, column: 2]` is more friendly than `JSON payload of the request is not valid`
I can agree to have the details in `cause` but we should rename this field to `details` then.
Finally, Throwable.getMessage is not supposed to include the classname, there's something wrong here that probably deserves to be fixed.

Reply:
The starting string `com.fasterxml.jackson.core.JsonParseException` is added by `toString()` from `JsonParseException.java`, not `getMessage()`. I didn't lookup carefully =.= , It had to be `JsonParseException.toString()` value in `cause` field. 

I will change `cause` to `details`, then update tests in other places. There are existing tests that assert `cause` field in other projects.

Reply:
This can be avoided by calling Exception::getMessage instead of the raw exception in ErrorResponder::Details